docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,24 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin.
+Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking certain events.  
 
-Data sources in Grafana supports both metrics and log data. The steps to build a logs data source plugin are largely the same as for a metrics data source. This guide assumes that you're already familiar with how to [Build a data source plugin](/tutorials/build-a-data-source-plugin/) for metrics.
+Grafana data source plugins support metrics, logs, and other data types. The steps to build a logs data source plugin are largely the same as for a metrics data source, but there are a few differences. 
```

Comment:
Instead of defining logs above, we could expand on _why_ there are the differences required compared to a metrics data source? (I cannot provide a suitable explanation here, but I'm sure someone else can). Alternatively we could end with "...but there are a few difference which we will explain in this guide."

Reply:
I've gone with your latter suggestion for now. When we have more background info, we can add it then. Created https://github.com/grafana/grafana/issues/67428